lib/task-data/tasks/catalog-bmc.js

```diff
@@ -8,57 +8,191 @@ module.exports = {
     implementsTask: 'Task.Base.Linux.Catalog',
     options: {
         commands: [
-            'sudo ipmitool lan print || true',
+            {
```

Comment:
Bummer, I guess we can't take advantage of mustache rendering loops because we have to represent these as objects, huh?